packages/core/src/authFlows/__tests__/auth.spec.ts

```diff
@@ -246,7 +246,7 @@ describe('Cognite Auth', () => {
         });
         expect(spiedLocationAssign).toBeCalledTimes(1);
         expect(spiedLocationAssign.mock.calls[0][0]).toMatchInlineSnapshot(
-          `"https://example.com/login/redirect?errorRedirectUrl=https%3A%2F%2Ferror-redirect.com&project=my-tenant&redirectUrl=https%3A%2F%2Fredirect.com"`
+          `"https://example.com/login/redirect?project=my-tenant&redirectUrl=https%3A%2F%2Fredirect.com&errorRedirectUrl=https%3A%2F%2Ferror-redirect.com"`
```

Comment:
They are the same, just query params changed order.

Reply:
why is the change needed then? 🤔

Reply:
Using URLSearchParams vs query-string which use different ordering rules

Reply:
And the matching here is on string level, it doesn't know this represents an URL

Reply:
Right!